contrib/devtools/github-merge.py

```diff
@@ -303,7 +303,12 @@ def main():
         message += subprocess.check_output([GIT,'log','--no-merges','--topo-order','--pretty=format:%H %s (%an)',base_branch+'..'+head_branch]).decode('utf-8')
         message += '\n\nPull request description:\n\n  ' + body.replace('\n', '\n  ') + '\n'
         try:
-            subprocess.check_call([GIT,'merge','-q','--commit','--no-edit','--no-ff','--no-gpg-sign','-m',message.encode('utf-8'),head_branch])
+            squash = input("Enter s for squash; anything else for regular merge commit: ").lower() == 's'
```

Comment:
why not use `ask_prompt` here like it's done everywhere else?

Reply:
good thought; I didn't see that :)